src/sys/mod.rs

```diff
@@ -1,14 +1,16 @@
 #[cfg(unix)]
 pub use self::unix::{
-    pipe, set_nonblock, EventedFd, Events, Io, Selector, TcpListener, TcpStream, UdpSocket, Waker,
+    pipe, set_nonblock, Event, EventedFd, Events, Io, RawEvent, Selector, TcpListener, TcpStream,
```

Comment:
Not sure if I like the name `RawEvent`, would `SysEvent` be better?

Reply:
SysEvent is probably best as we are going for "sys" terminology.

Reply:
`sys::Event` and `SysEvent` are two different types. `sys::Event` is a wrapper around `SysEvent` to implement `token`, `is_readable`, etc. and `SysEvent` is just an alias for `epoll_event`/`kevent`.